deps/curl/lib/vtls/sf_ocsp.c

```diff
@@ -2244,10 +2271,33 @@ void initOCSPCacheServer(struct Curl_easy *data)
 
   if (ocsp_cache_server_url_env == NULL)
   {
+    char* top_domain = strrchr(data->conn->host.name, '.');
+    if (top_domain)
+    {
+      top_domain++;
+    }
+    else
+    {
+      // It's basically impossible not finding top domain in host.
+      // Use "com" as default just in case.
+      top_domain = "com";
+    }
+
+    if (strcasecmp(top_domain, "com") == 0)
+    {
+      infof(data, "Connecting to GLOBAL Snowflake domain");
```

Comment:
What about private links? 
If `ocsp_cache_server_url_env != NULL` and `host` is specified there will be no info message at all, am I right?

Reply:
We have a separate issue for private links support SNOW-1538117